modules/local/nacho/qc/main.nf

```diff
@@ -1,10 +1,8 @@
 process NACHO_QC {
     label 'process_single'
 
-    conda "r-nacho=2.0.6 r-dplyr=1.2.2 r-ggplot2=3.4.4 r-fs=1.6.2 r-readr=2.1.5 r-tidyr=1.3.0"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/mulled-v2-9d27fb90b747ac2e521703d90daacce9cc1f33c5:98395a5d2e19da46499873cd2d76be73d6a0950d-0' :
-        'biocontainers/mulled-v2-9d27fb90b747ac2e521703d90daacce9cc1f33c5:98395a5d2e19da46499873cd2d76be73d6a0950d-0' }"
+    conda "${moduleDir}/environment.yml"
+    container "community.wave.seqera.io/library/r-dplyr_r-fs_r-ggplot2_r-nacho_pruned:9bb487ee68105a77"
```

Comment:
Why did you replace the original containers? And, why is there only one container instead of having a separate one for singularity?

Reply:
The current implementation of the conda directive did not work. Therefore, I had to create an `environment.yml` with conda versions for each software that worked. Given that these conda versions differed from the one in the mulled containers and to keep reproducibility between profiles, I had to replace the container directive. To make it have the same as the `environment.yml` and in line with current standards, I used Seqera containers. You now specify a single directive for both docker and singularity. 

Please see:
https://nf-co.re/blog/2024/seqera-containers-part-1

Reply:
Perfect, thanks for the clear explanation!